docs/t4_format_3d_detailed.md

```diff
@@ -161,7 +161,8 @@ pcd_data = pcd_data.reshape(-1, 5)
 - Almost equivalent to the [nuScenes format](https://www.nuscenes.org/nuscenes#data-format).
 - The [official tutorial notebook](https://www.nuscenes.org/nuscenes#tutorials) will help you get familiar with it.
 
-![nuscenes-schema.svg](nuscenes-schema.svg) from [the official site](https://www.nuscenes.org/nuscenes#data-format).
+![nuscenes-schema.svg](nuscenes-schema.svg)
+from [the official site](https://www.nuscenes.org/nuscenes#data-format).
```

Comment:
It still looks the same.. :thinking: 
![image](https://github.com/tier4/tier4_perception_dataset/assets/37187849/b8d6548b-d3ee-4167-ad33-13e9dbb52f0d)

Reply:
I'm sorry. I checked in vscode.
Now I fixed and confirmed in github ui.

<img width="1464" alt="2023-06-27_17-30" src="https://github.com/tier4/tier4_perception_dataset/assets/16977736/6abc687d-8909-4953-8ed0-1d3f1a9b2a1b">